MediaProcessor/cmake/test.cmake

```diff
@@ -22,8 +23,19 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(fmt)
 
+# Set platform-specific dynamic library for df
+if(APPLE)
+    set(DF_LIBRARY ${CMAKE_SOURCE_DIR}/lib/libdf.dylib)
```

Comment:
we don't yet support x86_64 on apple, which is unfortunate, but rosetta emulator just wouldn't work on my machine... So, we should keep consistent here with the src.cmake in terms of CPU check. Also no need for the comment on l26.